app/models/category.rb

```diff
@@ -0,0 +1,7 @@
+class Category < Classification
+  default_scope { where(:parent_id => 0) }
```

Comment:
I'm ok with this... The benefit of getting the API code to work as is justifies adding the class.